src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -129,11 +129,11 @@ export class HuiEntityCard extends LitElement implements LovelaceCard {
               >${"attribute" in this._config
                 ? stateObj.attributes[this._config.attribute!] ||
                   this.hass.localize("state.default.unknown")
-                : computeStateDisplay(
-                    this.hass.localize,
-                    stateObj,
-                    this.hass.language
-                  )}</span
+                : this.hass.localize(
+                    `component.${computeDomain(this._config.entity)}.state.${
+                      stateObj.attributes.device_class || "_"
+                    }.${stateObj.state}`
+                  ) || stateObj.state}</span
```

Comment:
Actually think that this is better

```suggestion
                : stateObj.attributes.unit_of_measurement
                ? stateObj.state
                : computeStateDisplay(
                    this.hass.localize,
                    stateObj,
                    this.hass.language
                  )}</span
```

Reply:
I just setup a new PC so I am still working on getting my DEV Env back up. But if we use StateDisplay then we can't style the measurement. So this is not better

Reply:
We only use statedisplay when there is no unit of measurement, so that would not interfere right?